html-aam/index.html

```diff
@@ -6429,6 +6429,71 @@ <h4 id="el-select-combobox">`select`<span class="el-context"> (with NO `multiple
             </tr>
           </tbody>
         </table>
+        <h4 id="el-selectedcontent">`selectedcontent`</h4>
+        <table aria-labelledby="el-selectedcontent">
+          <tbody>
+            <tr>
+              <th>HTML Specification</th>
+              <td>
+                <a data-cite="HTML">`selectedcontent`</a>
+              </td>
+            </tr>
+            <tr>
+              <th>[[wai-aria-1.2]]</th>
+              <td>
+                <div class="general"><a class="core-mapping" href="#role-map-generic">`generic`</a> role</div>
+              </td>
+            </tr>
+            <tr>
+              <th><a data-cite="core-aam-1.2/#roleMappingComputedRole">Computed Role</a></th>
+              <td class="role-computed"><div class="general">Use WAI-ARIA mapping</div></td>
+            </tr>
+            <tr>
+              <th>
+                <a href="https://msdn.microsoft.com/en-us/library/dd373608%28v=VS.85%29.aspx">MSAA</a> + <a href="http://accessibility.linuxfoundation.org/a11yspecs/ia2/docs/html/">IAccessible2</a>
+              </th>
+              <td>
+                <div class="general"><div class="general">Use WAI-ARIA mapping</div></div>
+              </td>
+            </tr>
+            <tr>
+              <th><a href="https://msdn.microsoft.com/en-us/library/ms726297%28v=VS.85%29.aspx">UIA</a></th>
+              <td>
+                <div class="general"><div class="general">Use WAI-ARIA mapping</div></div>
+              </td>
+            </tr>
+            <tr>
+              <th><a href="https://gnome.pages.gitlab.gnome.org/atk/">ATK</a></th>
+              <td>
+                <div class="general"><div class="general">Use WAI-ARIA mapping</div></div>
+              </td>
+            </tr>
+            <tr>
+              <th><a href="https://developer.apple.com/reference/appkit/nsaccessibility">AX</a></th>
+              <td>
+                <div class="general"><div class="general">Use WAI-ARIA mapping</div></div>
+              </td>
+            </tr>
+            <!-- <th><a href="#accessible-name-and-description-computation">Naming Algorithm</a></th> -->
+            <tr>
+              <th>Comments</th>
+              <td>
+                <p>
+                  The `selectedcontent` element is a declarable child of the `button` part of a custom styled `select` element. The `button` part of the `select` element's anatomy, and thus the
```

Comment:
this rewording should make this clearer...

```suggestion
                  The `selectedcontent` element is to be used as a descendant of the `button` part of a custom styled `select` element. The `button` part of the `select` element's anatomy, and thus the
```